src/main/java/org/dswarm/graph/hash/HashUtils.java

```diff
@@ -16,12 +16,18 @@
  */
 package org.dswarm.graph.hash;
 
+import com.github.emboss.siphash.SipKey;
+
 /**
  *
  */
 public final class HashUtils {
 
-	private HashUtils() {}
+	public static final SipKey	SPEC_KEY	= new SipKey(HashUtils.bytesOf(0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0a, 0x0b,
+													0x0c, 0x0d, 0x0e, 0x0f));
+
```

Comment:
from http://www.forward.com.au/pfod/SipHashJavaLibrary/ I would tend to say that is simply defines the format for the byte array, i.e., BigEndian or LittleEndian